Engine/Settings.cs

```diff
@@ -52,13 +52,13 @@ public Settings(object settings, Func<string, string> presetResolver)
                 throw new ArgumentNullException(nameof(settings));
             }
 
-            includeRules = new List<string>();
-            excludeRules = new List<string>();
-            severities = new List<string>();
-            ruleArguments = new Dictionary<string, Dictionary<string, object>>(StringComparer.OrdinalIgnoreCase);
-            var settingsFilePath = settings as string;
+            this.includeRules = new List<string>();
```

Comment:
If a change like this is made, I think it may as well be a change to `_includeRules` instead. That's admittedly more of a style opinion, but it's consistent with other PowerShell code and a less verbose way of designating a private field.